Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -68,11 +68,25 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath, b
 if ~isfield(x,'Q')
     x.Q = struct;
 end
+
+if ~isfield(x,'MagneticFieldStrength') || isempty(x.MagneticFieldStrength)
```

Comment:
This parameter should probably be in `x.Q` together with the other quantification parameters?

Reply:
Also, I wonder if this location alone is sufficient. MagneticFieldStrength is a BIDS parameter, so if it is missing then the user should be warned (also) earlier?

Reply:
Yes. User will be warned before, but for completeness and modularity, I kept it here as well. Also, I wanted to be minimally invasive - all previous defaults were set to 3T, so I made 3T the default here also to avoid changes in the results